libc/newhdrgen/yaml/sys/mman.yaml

```
@@ -79,37 +79,37 @@ functions:
       - type: void *
       - type: size_t
   - name: mlockall
-    standards: 
+    standards:
       -POSIX
```

```suggestion
      - POSIX
```

I don't know if you want to turn this into a more general cleanup PR or not.

---

I'll commit this, but we may make a new pr for general yaml cleanup.